### DISCUSSION TITLE
Vercel Domain CAA record

### DISCUSSION THREAD

**Discussion by phpmaps** [vercel/vercel#5189] [Help] [answered]
I need to point my domain name managed in Vecel to an app that is hosted in Azure. 

 So far the DNS is properly resolving using A and TXT records, but SSL is broken :(  

 **Two questions:** 

1)  Is there a way to access the Vercel cert for my domain as a PFX, from my Domain dashboard? 

2)  How do i delete my default CAA DNS record so that I may be able to use another cert?

When I try to delete the existing CAA record it says "default records cannot be deleted".  This seems to be an issue, because having that CAA record along side my domain seems to block me from from registering any other SSL cert for this domain.  Any guidance would be appreciated.   Here's a quick link explained there error i see when trying to register a different SSL cert :(

https://www.godaddy.com/help/certificate-authority-authorization-caa-record-errors-27268

ERROR
A DNS CAA record exists for domain(s) coolexample.com which forbids the issuance of this certificate.	

RESOLUTION
Contact your DNS provider to have the CAA record removed. Once this is gone, request the certificate again.

**Comment by judewang** [comment]
I do have the same problem. Firebase Hosting is migrating their SSL certificate provider to a new, Google-run Certificate Authority (CA). Vercel's CAA records will prevent the migration.

**Comment by paulogdm** [comment, marked as answer]
You can add a new CAA record to your root domain. An example: https://www.whatsmydns.net/#NS/vercel.com


After you do that, the domain will work as expected.

**Reply by judewang** [reply]
@paulogdm Thank you!

**Comment by elitan** [comment]
@paulogdm What if I only need to delete the current CAA record? How do I do that?

**Reply by paulogdm** [reply]
You cannot delete the default CAA record. If you do that, your site may receive errors in modern browsers.

**Reply by elitan** [reply]
I want to use my own SSL certificate. It's for a domain that I host outside Vercel. How can I do that?

**Reply by paulogdm** [reply]
You can include another CAA record. It will work as expected: https://serverfault.com/questions/972720/can-i-create-a-caa-record-for-all-sub-domains

**Reply by elitan** [reply]
Maybe I wasn't clear enough: How do I **delete** the CAA record? Not replace/edit it.
